### PR TITLE
Add machine id to kurl reporting

### PR DESCRIPTION
**Pull request by emosbaugh**

### CODE REVIEW

File: pkg/cli/machineid.go [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import (
+	"fmt"
+
+	"github.com/replicatedhq/kurl/pkg/machineid"
+	"github.com/spf13/cobra"
+)
+
+func NewMachineidCmd(cli CLI) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "machineid",
```
Comment (emosbaugh) [inline pkg/cli/machineid.go:12]: change this to hostid

**Comment by emosbaugh** [comment]
how about this for the cli?

```
ethan@repldev-ethan-new:~/go/src/github.com/replicatedhq/kurl$ ./build/bin/kurl host
Perform operations on the kURL host

Usage:
  kurl host [command]

Available Commands:
  preflight   Runs kURL host preflight checks
  protectedid Prints the kURL host protected machine id

Flags:
  -h, --help   help for host

Global Flags:
      --debug   enable debug logging

Use "kurl host [command] --help" for more information about a command.
ethan@repldev-ethan-new:~/go/src/github.com/replicatedhq/kurl$ ./build/bin/kurl host protectedid
3578997facfa71a2c3cdedfa5b0ddb0db4a093b2a93bddbbb24f29820bf8a0e7
```

File: scripts/common/common.sh [hunk 2]
```diff
@@ -697,3 +697,8 @@ function maybe() {
 
     $cmd $args 2>/dev/null || true
 }
+
+MACHINE_ID=
+function get_machine_id() {
+    MACHINE_ID="$("${DIR}"/bin/kurl machineid || true)"
```
Comment (areed) [inline scripts/common/common.sh:703]:
```suggestion
    MACHINE_ID="$(${DIR}/bin/kurl host protectedid || true)"
```

Comment (emosbaugh) [reply scripts/common/common.sh:703]: why?
Screen Shot 2021-03-04 at 2 28 50 PM

Comment (emosbaugh) [reply scripts/common/common.sh:703]: thanks

**Review by areed — APPROVED** [review]

**Pull request merged**